cypress/support/commands.js

```diff
@@ -31,6 +31,24 @@ Cypress.Commands.add('logout', (route = '/') => {
 	})
 })
 
+Cypress.Commands.add('createFolder', (user, target) => {
+	cy.login(user)
```

Comment:
I'd just assume you're logged in here.

Reply:
I'd keep this for now to avoid that the command call could interfere with previously called logins. Feels like those could be harder to debug and I think the additional check shouldn't hurt too much.

nevertheless those should go into the lib at some point